unabridged-works-volume-1/original/13-a-question-proposed.adoc

```diff
@@ -276,7 +289,8 @@ and sought with our hearts to prevent thy drinking of that bitter cup,
 which is to go round the nations.
 And if thine ear could have been opened, thou mightest have been spared.
 
-=== Postscript
+[.postscript]
```

Comment:
Yeah, this is a bit confusing, there are a few of these scattered in all our books, but not many.  Anyway, here since it's not really a postcript to a letter or epistle, and it kind of introduces a new _section_ I think i would just leave it as a heading called Postscript. So basically, change it back to how it was 👍